docs/tables/azuread_security_defaults_policy.md

```diff
@@ -0,0 +1,29 @@
+# Table: azuread_security_defaults_policy
+
+Security defaults in Azure Active Directory (Azure AD) make it easier to be secure and help protect your organization. Security defaults contain pre-configured security settings for common attacks.
+
+## Examples
```

Comment:
Are there any additional examples we can use? The table seems a little light if not.

Reply:
@cbruno10 Unfortunately the table does not have much property so we can use those in examples. The response of this table only indicates whether the pre-configured security settings as configured or not.
This table requires in one of the Microsoft 365 compliance control:
```
Ensure Security Defaults is disabled on Azure Active Directory

Remediation:
  To disable security defaults in your directory:
    1. Sign in to the Azure portal as a security administrator, Conditional Access administrator, or global administrator.
    2. Browse to Azure Active Directory > Properties.
    3. Select Manage security defaults.
    4. Set the Enable security defaults toggle to No.
    5. Select Save.
```